crates/bevy_ui/src/focus.rs

```diff
@@ -55,6 +56,22 @@ impl Default for Interaction {
     }
 }
 
+/// Holds the last interaction that occurred for a UI node.
+///
+/// This is commonly used by adding this component
+/// to a UI node such that [`Click`] events are sent on click/press-release action.
+#[derive(Component, Default, Debug, Clone)]
+pub struct LastInteraction(Interaction);
+
+/// Used to publish which entity was clicked
+///
+/// Commonly used by creating a UI node that contains [`LastInteraction`] and [`Interaction`] components
+/// and then using [`EventReader<Click>`] to obtain the list of clicked UI nodes.
```

Comment:
Looks like doctests failing because it can't find `EventReader`, let's specify the scope:
```suggestion
/// and then using [`EventReader<Click>`](event::EventReader) to obtain the list of clicked UI nodes.
```